webtransport/quic/client-indication.any.js

```diff
@@ -0,0 +1,32 @@
+// META: quic=true
+// META: script=/common/get-host-info.sub.js
+
+const PORT = 8983;
+const HOST = get_host_info().ORIGINAL_HOST;
+const ORIGIN = get_host_info().ORIGIN;
+const BASE = `quic-transport://${HOST:${PORT}/handlers`;
+
+promise_test(async (test) => {
+  function onClosed() {
+    assert_unreached('The closed promise should be ' +
+                     'fulfilled or rejected after getting a PASS signal.');
+  }
+  t = new QuicTransport(`${BASE}/client-indication.quic.py?origin=${ORIGIN}`);
```

Comment:
`const t`? Although I would prefer `qt`, as `t` seems a little **too** succinct, and I normally use `t` for the test object.

Reply:
Done.